### DISCUSSION TITLE
Decrease padding in project cards

### DISCUSSION THREAD

**Discussion by AlesBucek** [alshedivat/al-folio#2210] [Q&A] [answered]
Firstly, thanks a lot for making freely available such a great template and thanks everybody who is providing advice on customizing it!

I want to customize the padding of project cards. Currently, the padding is rather large (at least to my taste, screenshot attached) and the title field under the image takes massive amount of space which I'd rather dedicate to the image and this becomes even more pronounced when >3 cards are rendered per row. I've explored "projects.liquid" and "_base.scss" files where I could customize the card width, first letter capitalization, title font size, category separator lines but I can not figure out how to customize the padding. Thanks in advance for any hints.
<img width="901" alt="Screenshot 2024-02-17 at 21 30 37" src="https://github.com/alshedivat/al-folio/assets/12627487/576d6446-ae10-44ff-9da0-c2d3f3d47deb">

**Comment by george-gca** [comment]
The padding should be done in `_base.scss`. The best way for you to try in advance where you could add the change is by using [chrome dev tools](https://developer.chrome.com/docs/devtools/css) or [firefox dev tools](https://firefox-source-docs.mozilla.org/devtools-user/).

**Reply by AlesBucek** [reply, marked as answer]
Thanks, I was not familiar with the dev tools. I found it - it is in `projects.liquid` file, modification of the style of `class="card-body"`
